package.json

```diff
@@ -23,6 +24,9 @@
   "devDependencies": {
     "@dosomething/babel-config": "^1.0.0",
     "@dosomething/webpack-config": "^2.0.4",
+    "babel-jest": "^21.0.2",
+    "jest": "^21.0.2",
+    "regenerator-runtime": "^0.11.0",
```

Comment:
@DFurnes [This](https://facebook.github.io/jest/docs/en/getting-started.html#content) said to also install `regenerator-runtime` but I don't think PN has it - is this needed?

Reply:
Nope, we should be using npm 4+ everywhere so no need.

Reply:
Ok cool - deleted!